libs/barista-components/chart/src/selection-area/selection-area.ts

```diff
@@ -128,7 +128,8 @@ import {
     '[attr.tabindex]': '0',
   },
 })
-export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
+// tslint:disable-next-line: class-name
+export class _DtChartSelectionArea implements AfterContentInit, OnDestroy {
```

Comment:
Why do we need to rename the className as it is exported with this name?

Reply:
The `ts-api-guard` script cannot deal with renames in exports and fails. That's why I renamed the class to prevent the renaming of the export.